impl/src/bin/cargo-raze.rs

```diff
@@ -66,6 +68,11 @@ Options:
 
 fn main() -> Result<()> {
   let options: Options = Docopt::new(USAGE)
+    .map(|d| {
+      d.version(Some(
+        concat!("cargo-raze ", env!("CARGO_PKG_VERSION")).to_string(),
+      ))
```

Comment:
does this cause an automatic exit?

Reply:
(when seeing the version flag)

Reply:
Yep. As with `--help`, there’s a [`Version` error variant][verr], so the
program hits the `|e| e.exit()`.

[verr]: https://docs.rs/docopt/1.1.0/docopt/enum.Error.html#variant.Version